src/components/CommercesAreaItem.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { View, Text, TouchableHighlight } from 'react-native';
+import { NavigationActions } from 'react-navigation';
+import { Card } from 'react-native-elements';
+import { connect } from 'react-redux';
+import { Ionicons } from '@expo/vector-icons';
+
+class CommercesAreaItem extends Component {
+  onButtonPressHandler(id) {
+    const navigateAction = NavigationActions.navigate({
+      routeName: 'commercesList',
+      params: { idArea: id }
+    });
+
```

Comment:
esto se puede hacer asi para no tener que importar NavigationActions:

this.props.navigation.navigate('commercesList', { idArea: id })